src/client.js

```diff
@@ -1777,6 +1777,34 @@ class Client {
         const result = this._toRepresentation(doc);
         return result;
     }
+
+    /**
+     * Creates and returns a new instance of the specified entity
+     * @param entityType
+     * @param entity
+     * @returns {Promise<XML.XtkObject>}
+     */
+    async newInstance(entityType, entity) {
+        try{
+            const that = this;
+            const root = entityType.split(":").pop();
+            const xml= this._fromRepresentation(root, entity, 'SimpleJson')
```

Comment:
After discussion, could you:
- make sure first parameter is a schemaId, i.e. "nms:delivery" instead of "xtk:persist|nms:delivery"
- make sure the function implementation makes the soap call with  "xtk:persist|nms:delivery" URN
- make sure the function implementation adds the "xtkschema" attribute